### PR TITLE
Adding note about login slug to Auth0

### PR DESCRIPTION
**Pull request by matthewshaver**
## What are you changing in this pull request and why?

Adding the requested login slug disclaimer per internal slack

## Checklist

- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [x] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

File: website/docs/docs/cloud/manage-access/auth0-migration.md [hunk 1]
```diff
@@ -32,6 +32,13 @@ Once you have opted to begin the migration process, the following steps will var
 
 SAML 2.0 users must update a few fields in the SSO app configuration to match the new Auth0 URL and URI.  You can approach this by editing the existing SSO app settings or creating a new one to accommodate the Auth0 settings. One approach isn't inherently better, so choose whichever works best for your organization.
 
+:::warning Login {slug}
+
+Your login slug should not contain an underscore. If it does, it must be changed in the account settings.
```
Comment (mirnawong1) [inline website/docs/docs/cloud/manage-access/auth0-migration.md]: changing to active voice

```suggestion
The login slug shouldn't have an underscore. If it does, you need to change it on the **Account Settings** page. After changing the slug, the admins should share it with dbt Cloud users, as the login URL will be different.
```

File: website/docs/docs/cloud/manage-access/auth0-migration.md [hunk 2]
```diff
@@ -32,6 +32,13 @@ Once you have opted to begin the migration process, the following steps will var
...
+Once the slug is changed, have your admins share it with dbt Cloud users, as the login URL will change.
```
Comment (mirnawong1) [inline website/docs/docs/cloud/manage-access/auth0-migration.md]:
```suggestion
```

**Comment by mirnawong1** [comment]
hey @matthewshaver left a suggestion to switch to active voice - let me know what you think!

File: website/docs/docs/cloud/manage-access/auth0-migration.md [hunk 3]
```diff
@@ -32,6 +32,12 @@ Once you have opted to begin the migration process, the following steps will var
 
 SAML 2.0 users must update a few fields in the SSO app configuration to match the new Auth0 URL and URI.  You can approach this by editing the existing SSO app settings or creating a new one to accommodate the Auth0 settings. One approach isn't inherently better, so choose whichever works best for your organization.
 
+:::warning Login {slug}
+
+The login slug shouldn't have an underscore. If it does, you need to change it on the **Account Settings** page. After changing the slug, the admins should share it with dbt Cloud users, as the login URL will be different.
```
Comment (matthewshaver) [inline website/docs/docs/cloud/manage-access/auth0-migration.md]:
```suggestion
The login slug shouldn't have an underscore. If it does, you need to change it on the **Account Settings** page. After changing the slug, the admins must share it with dbt Cloud users, as the login URL will differ.
```

**Review by mirnawong1 — APPROVED** [review]
nice tweak, Matt! Lgtm 🥳

**Pull request merged**